SwrveConversationSDK/src/main/java/com/swrve/sdk/conversations/ui/ConversationFragment.java

```diff
@@ -310,14 +310,14 @@ private void applyMultiValueInputStyles(MultiValueInput content) {
         }
 
         for (ChoiceInputItem item : content.getValues()) {
-            Typeface itemTypeface = getTypeface(titleStyle, SwrveConversationConstants.DEFAULT_MVI_OPTION_TYPEFACE);
             if (item.getStyle() == null) { // v1,v2, v3 of conversations have no ChoiceInputItem styles
                 ConversationStyle itemStyle = new ConversationStyle(0, "", titleStyle.getBg(), titleStyle.getFg(), null);
                 itemStyle.setFg(titleStyle.getFg());
                 itemStyle.setTextSize(getResources().getDimensionPixelSize(R.dimen.swrve__conversation_mvi_option_default_text_size));
-                itemStyle.setTypeface(itemTypeface);
+                itemStyle.setTypeface(titleTypeface); // use the same typeface as title
```

Comment:
Old v1/v2/v3 conversations don't have any style inside the choice elements. However the parent title did have a style. This code will populate the child elements with that same parent style.